webapp/components/wpt-header.js

```diff
@@ -75,6 +75,7 @@ class WPTHeader extends WPTFlags(PolymerElement) {
       <nav>
         <!-- TODO: handle onclick with wpt-results.navigate if available -->
         <a href="/">Latest Run</a>
+        <a href="/interop[[path]]?[[query]]">Interoperability</a>
```

Comment:
Let's move this to between 'Insights' and 'Processor'.

(Ultimately I think we'll want to shuffle things around, but for now this seems a bit more coherent than Latest / Interop / Recent / Insights / Processor. Plus we know people rarely use this view anyway.)